build/parents/integrationtest/pom.xml

```diff
@@ -72,348 +58,6 @@
                         </execution>
                     </executions>
                 </plugin>
-                <plugin>
-                    <groupId>io.fabric8</groupId>
-                    <artifactId>docker-maven-plugin</artifactId>
-                    <version>${version.docker.maven.plugin}</version>
-                    <configuration>
-                        <skip>${test.containers.run.skip}</skip>
```

Comment:
I think you need to remove this property and all references to it (in GitHub Actions workflows, in CONTRIBUTING.md, ...)